datasets/common_voice/common_voice.py

```diff
@@ -0,0 +1,743 @@
+# coding=utf-8
+# Copyright 2021 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Common Voice Dataset"""
+
+from __future__ import absolute_import, division, print_function
+
+import os
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{commonvoice:2020,
+  author = {Ardila, R. and Branson, M. and Davis, K. and Henretty, M. and Kohler, M. and Meyer, J. and Morais, R. and Saunders, L. and Tyers, F. M. and Weber, G.},
+  title = {Common Voice: A Massively-Multilingual Speech Corpus},
+  booktitle = {Proceedings of the 12th Conference on Language Resources and Evaluation (LREC 2020)},
+  pages = {4211--4215},
+  year = 2020
+}
+"""
+
+_DESCRIPTION = """\
+Common Voice is Mozilla's initiative to help teach machines how real people speak.
+The dataset currently consists of 7,335 validated hours of speech in 60 languages, but we’re always adding more voices and languages.
+"""
+
+_HOMEPAGE = "https://commonvoice.mozilla.org/en/datasets"
+
+_LICENSE = "https://github.com/common-voice/common-voice/blob/main/LICENSE"
+
+_LANGUAGES = {
+    "ab": {
+        "Language": "Abkhaz",
+        "Date": "2020-12-11",
+        "Size": "39 MB",
+        "Version": "ab_1h_2020-12-11",
+        "Validated_Hr_Total": 0.05,
+        "Overall_Hr_Total": 1,
+        "Number_Of_Voice": 14,
+    },
+    "ar": {
+        "Language": "Arabic",
+        "Date": "2020-12-11",
+        "Size": "39 MB",
```

Comment:
I think this parameter and the following ones still have to be corrected for every language

Reply:
I updated the data with correct information in the latest pull request.